docs/EMERGENCY.md

```diff
@@ -0,0 +1,17 @@
+# Emergency mode
+
+ES-Operator will not operate a non-green cluster. Therefore, there are certain situations that require human intervention to recover an Elasticsearch cluster running with ES-Operator.
+
+## Stopping the bleeding
+
+The first and foremost goal should be to get your cluster back into green state. The [cluster reroute API](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-reroute.html#cluster-reroute-api-request-body) can be used to temporarily allocate an empty primary, until you have recovered your data.
```

Comment:
@otrosien I think this section could benefit from more context/details about what's being proposed and why.

Reply:
hm.. the context is: cluster non-green, as described above. The why is there as well, es-operator doesn't touch non-green ES clusters.

Reply:
of course cluster-reroute is only one means to recover the cluster state. Should we mention others? (deleting the broken index, recovering from a snapshot...)